src/main.cpp

```diff
@@ -593,6 +606,11 @@ int main(int argc, char *argv[])
 
     Log::set_max_level(config.debug_log_level);
 
+    Endpoint::sniffer_sysid = config.sniffer_sysid;
```

Comment:
I think, thing should better be at `Mainloop::add_endpoints()` since the main.cpp is only doing config parsing and not dealing with the endpoints themselves.
Also only change the endpoint variable when `config.sniffer_sysid != 0` since it was initialized with 0 (similar to [here](https://github.com/mavlink-router/mavlink-router/blob/fa0f260a3ad6372c5f8c0f212a0295e11b58d3ff/src/mainloop.cpp#L421))

Reply:
done